docs/features.md

```diff
@@ -251,7 +251,10 @@ process execution, display the reason for trigger and the associated context.
 
 ## `gef-remote` debugging ##
 
-It is possible to use `gef` in a remote debugging environment.
+It is possible to use `gef` in a remote debugging environment. Required files
+will be automatically downloaded and cached in a temporary directory (`/tmp/gef` on most
+Unix systems). Remember to manually delete the cache if you change the target file or
```

Comment:
I believe it is better to rely on `gdb.events` instead. That is the only way to trigger some actions when the target binary has changed, and **only when** it has changed.